README.md

```diff
@@ -23,7 +23,7 @@ Once you have created or retrieved credentials for your App Store account, perfo
 
 1. Install the App Store extension from the [VSTS Marketplace](https://marketplace.visualstudio.com/items/ms-vsclient.app-store).
 
-2. Go to your Visual Studio Team Services or TFS project, click on the **Build** tab, and create a new build definition (the "+" icon) that is hooked up to your project's appropriate source repository.
+2. Go to your Azure DevOps or TFS project, click on the **Build** tab, and create a new build definition (the "+" icon) that is hooked up to your project's appropriate source repository.
```

Comment:
I think you can just remove 'of TFS project'.  'Azure DevOps' should cover cloud and server, no?